Tests/LibWeb/AccessibilityBridge/harness/tree.py

```
@@ -0,0 +1,217 @@
+        text_repr = repr(text[:60]) if text else ""
+        lines.append(f"{indent}{role} name={name!r} text={text_repr}")
+
+    walk(root, _visit, max_depth=max_depth)
+    return "\n".join(lines)
+
+
+def get_attributes_dict(obj: Atspi.Accessible) -> dict:
+    """Return object attributes (e.g., "tag", "xml-role", "level") as a dict."""
+    try:
+        attrs = Atspi.Accessible.get_attributes(obj)
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)